themes/default/content/learn/pulumi-fundamentals/create-a-pulumi-project/index.md

```diff
@@ -0,0 +1,138 @@
+---
+title: "Creating a Pulumi Project"
+layout: topic
+date: 2021-09-10T12:00:00-05:00
+draft: true
+description: Create your first Pulumi project.
+meta_desc: Create your first Pulumi project.
+index: 0
+estimated_time: 5
+meta_image: meta.png
+authors:
+    - sophia-parafina
+    - laura-santamaria
+tags:
+    - fundamentals
+    - projects
+    - docker
+links:
+    - text: Code Repo
+      url: https://github.com/pulumi/tutorial-pulumi-fundamentals
+block_external_search_index: true
+---
+
+Infrastructure in Pulumi is organized into
+[_projects_](https://www.pulumi.com/docs/reference/glossary/#project). In the 
```

Comment:
why are we italicizing these names?

Reply:
We're trying to call out jargon in an industry-standard way (e.g., see _Terms (New)_ at https://docs.rackspace.com/docs/style-guide/style/text-formatting, which was based on sources like the Microsoft Technical Style Guide). A bit of an old editing habit, really.

Reply:
if they werent links i would prolly support this, but its a bit strange for some links to get that style and some not.

Reply:
Yep, I do like this (italicizing, I mean; irrespective of linking), and it's common practice when introducing terms like this -- I remember it from the Manning Style Guide as well, which I believe is largely based on the Chicago Manual of Style. We also do this in some of our other introductory pages -- e.g., here: https://www.pulumi.com/docs/intro/concepts/ So IMO, this is OK. 👍

Reply:
Ah, I see. The intro pages don't link to the glossary for the italics. How about I remove the links to the glossary since I explain the terms anyway? I had wanted to utilize that glossary more since we have it, but if we're not using it as much in the other intro pages, let's keep to the same experience. Fixes incoming.

Reply:
That sounds good!

Reply:
Ok, all fixed!